src/utils/graphql.ts

```diff
@@ -0,0 +1,27 @@
+import { InMemoryCache, NormalizedCacheObject } from 'apollo-cache-inmemory';
```

Comment:
Just FYI, for `getServerSideProps` the usage of apollo might change, see:  https://github.com/lfades/next-with-apollo/issues/114

Reply:
Thanks, interesting. I also learned that the current implementation opt-in with SSR for all pages, due to the usage of `getDataFromTree`.

See https://github.com/zeit/next.js/issues/12476

Reply:
Great =)

Reply:
Not really ahah, I meant it disables auto static optimization by doing so. 😅

Reply:
lol I meant, "got it" haha